src/common/dependency-manager/local-manager.ts

```diff
@@ -5,12 +5,13 @@ import path from 'path';
 import DependencyManager, { DependencyNode, EnvironmentConfigBuilder, ServiceConfig, ServiceConfigBuilder, ServiceNode } from '../../dependency-manager/src';
 import IngressEdge from '../../dependency-manager/src/graph/edge/ingress';
 import GatewayNode from '../../dependency-manager/src/graph/node/gateway';
+import { Dictionary } from '../../dependency-manager/src/utils/dictionary';
 import { readIfFile } from '../utils/file';
 import PortUtil from '../utils/port';
 import LocalDependencyGraph from './local-graph';
 import { LocalServiceNode } from './local-service-node';
 
-declare type LinkedServicesMap = { [serviceName: string]: string };
+export declare type LinkedServicesMap = Dictionary<string>;
```

Comment:
ha I meant the type is unnecessary, but its fine.

Reply:
I thought so too but the wording made it sound like you wanted to keep it. Removed in the liveness probe PR